release-please-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "separate-pull-requests": true,
+  "packages": {
+    "infra/blueprint-test": {
```

Comment:
Lets also set `bump-minor-pre-major` to true as we havent defined what 1.0 looks like yet.

Reply:
SG